src/server/gql/prisma.ts

```diff
@@ -0,0 +1,16 @@
+import { PrismaClient } from '@prisma/client'
+import { PRISMA_LOG_LEVELS } from '../environment'
+
+const log = PRISMA_LOG_LEVELS.split(',').filter(Boolean) as Array<
+  'query' | 'info' | 'warn' | 'error'
+>
```

Comment:
**Nitpick**

Seems like it might be simpler to do this:

```suggestion
const log = ['query', 'info', 'warn', 'error'] as const
```

Reply:
I want to be able to control this from the logs. So I want to be able to provide only a subset,for debugging.